images/builder/main.go

```diff
@@ -150,8 +186,9 @@ func runSingleJob(o options, jobName, uploaded, version string, subs map[string]
 
 	cmd := exec.Command("gcloud", args...)
 
+	var p string
```

Comment:
If this is surviving outside of scope, please give it a descriptive name. `jobPath`, for example.

Reply:
Will do. :)